superset-frontend/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -605,7 +605,7 @@ class SqlEditor extends React.PureComponent {
             }}
             key="2"
           >
-            {t('CREATE VIEW AS')}
+            {t('Create view as')}
```

Comment:
CREATE VIEW AS is an outlier in this situation. We want to mimic the SQL language in this case so we're displaying this text as all caps. In this instance the correct form should be 'CREATE VIEW AS'